query/sources/common/src/main/java/org/codice/ddf/admin/sources/utils/RequestUtils.java

```diff
@@ -164,29 +189,85 @@ public ReportImpl endpointIsReachable(UrlField urlField) {
         return report;
     }
 
-    private WebClient generateClient(String url, CredentialsField creds,
-            Map<String, String> queryParams) {
-        SecureCxfClientFactory<WebClient> clientFactory = createFactory(url, creds);
+    public class WebClientBuilder {
```

Comment:
I don't think this will matter since the ctors are private and this class should only be instantiated through the creation methods RequestUtils provides.